src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -821,13 +858,13 @@ protected TNode AddError<TNode>(TNode node, int offset, int length, ErrorCode co
         protected TNode AddErrorToFirstToken<TNode>(TNode node, ErrorCode code) where TNode : CSharpSyntaxNode
         {
             var firstToken = node.GetFirstToken();
-            return WithAdditionalDiagnostics(node, MakeError(firstToken.GetLeadingTriviaWidth(), firstToken.Width, code));
+            return WithAdditionalDiagnostics(node, MakeError(offset: 0, firstToken.Width, code));
```

Comment:
these helpers all got much simpler conceptually.  We're placing the diag on the node/token.  So the offset is trivially 0.